lib/evss/disability_compensation_form/form4142_processor.rb

```diff
@@ -49,12 +49,14 @@ def generate_stamp_pdf
         pdf = PdfFill::Filler.fill_ancillary_form(
           @submission.form[Form526Submission::FORM_4142], @submission.submitted_claim_id, FORM_ID
         )
-        stamped_path = CentralMail::DatestampPdf.new(pdf).run(text: 'VA.gov', x: 5, y: 5)
+        stamped_path = CentralMail::DatestampPdf.new(pdf).run(text: 'VA.gov', x: 5, y: 5,
+                                                              form_submission_date: @submission.created_at)
```

Comment:
In order to address Emily's concerns about the Central Mail DateStamp functionality, I'm adding in an optional `form_submission_date` attribute to the settings hash. If it's passed in, the Central Mail DateStampPdf module will use that to set the date on the PDF timestamp instead of `Time.zone.now`.